packages/datepicker-react/src/internal/reducer.ts

```diff
@@ -19,11 +19,17 @@ interface ValuePropChangedAction {
     payload: Date | undefined;
 }
 
+interface ValidateOnblur {
```

Comment:
```suggestion
interface ValidateOnBlur {
```

Men er ikke helt sikker på at det er riktig navn, for er vel strengt tatt ikke det vi gjør? Er vel heller SET_VALUE_ON_BLUR?